bpython/curtsies.py

```diff
@@ -122,6 +122,9 @@ def after_suspend():
                 window.__enter__()
                 interrupting_refresh()
 
+            def get_top_usable_line():
+                return window.top_usable_row
+
```

Comment:
I'd be up for trying this refactor in a new branch, I'll open a PR early so it's easy to talk about it.